fe/fe-core/src/main/java/com/starrocks/sql/optimizer/cost/HashJoinCostModel.java

```diff
@@ -99,7 +102,10 @@ public double getCpuCost() {
                 buildCost = rightOutput;
                 probeCost = leftOutput;
         }
-        return buildCost + probeCost;
+        double joinCost = buildCost + probeCost;
+        // should add output cost
+        joinCost += joinStatistics.getComputeSize();
```

Comment:
would it affect existing cases ?

Reply:
yes, it changes some ut plans. But I think it is reasonable.

Reply:
What is the reason for increasing the cost of this part? Is the cost is a linear relationship with outputsize? If the join output result is too large, the overall cost may increase too much and affect the plan selection.

Reply:
because after mv rewrite, join may be rewritten as a mv scan operator. the join's cost should compare with rewritten scan mv operator, which should consider the output size of join.